azure/sdk-identity-resources-storage/Program.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Threading.Tasks;
+
+using Azure;
+using Azure.Identity;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.Resources.Models;
+using Azure.ResourceManager.Storage;
+using Azure.ResourceManager.Storage.Models;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+
+using AccessTier = Azure.ResourceManager.Storage.Models.AccessTier;
+using Sku = Azure.ResourceManager.Storage.Models.Sku;
+
+namespace AzureIdentityStorageExample
+{
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            string subscriptionId = Environment.GetEnvironmentVariable("AZURE_SUBSCRIPTION_ID");
+
+            DefaultAzureCredential credentials = new DefaultAzureCredential();
+
+            ResourcesManagementClient resourcesManagementClient = new ResourcesManagementClient(subscriptionId, credentials);
+            StorageManagementClient storageManagementClient = new StorageManagementClient(subscriptionId, credentials);
+
+            // Create a Resource Group
+            string resourceGroupName = await CreateResourceGroup(resourcesManagementClient);
+
+            // Create a Storage account
+            string storageName = await CreateStorageAccount(storageManagementClient, resourceGroupName);
+
+            // Create a container and upload a blob
+            await UploadBlobToStorageAccountUsingClientConnectionString(storageManagementClient, resourceGroupName, storageName);
```

Comment:
```suggestion
            await UploadBlobToStorageAccounAsync(storageManagementClient, resourceGroupName, storageName);
```

Simplify name (omit parameter/argument details in name), and include `Async` suffix